Products/CMFPlomino/PlominoForm.py

```diff
@@ -1335,6 +1335,37 @@ def validation_errors(self, REQUEST):
                     errors=json.dumps({'success': True}))
 
 
+    security.declarePrivate('_get_js_hidden_subforms')
+    def _get_js_hidden_subforms(self, REQUEST, doc, validation_mode=False):
+        hidden_forms = []
+        hidewhens = json.loads(
```

Comment:
I copied the functionality of `_get_js_hidden_fields`, which operates the same way.

If I understand you correctly, I should create a new `getHidewhen` function which returns the hidewhens as a python object. Then `getHidewhenAsJSON` would use that new function internally as well?

That makes sense, but would we also then use `getHidewhen` in `_get_js_hidden_fields`? Surely then the function should just be `get_hidden_fields`?